src/server/rdb_save.cc

```diff
@@ -1333,6 +1333,8 @@ size_t RdbSaver::Impl::GetTotalBuffersSize() const {
 
   auto cb = [this, &channel_bytes, &serializer_bytes](ShardId sid) {
     auto& snapshot = shard_snapshots_[sid];
+    if (!snapshot)
```

Comment:
@BorysTheDev  please add a comment explaining how come RdbSave::Impl with shard_snapshots_ exist but they are not initialized yed.